modules/ROOT/pages/sync/data_sources.adoc

```diff
@@ -0,0 +1,58 @@
+include::{partialsdir}/attributes.adoc[]
+
+//':context:' is a vital parameter. See: http://asciidoctor.org/docs/user-manual/#include-multiple
+:context: data_sources
+
+[id='{context}_data_sources']
+= Managing your data sources using the {sync-service} UI
```

Comment:
This does currently not appear in nav.adoc, should it? Or maybe add the content directly to data_sync_ui.adoc?

Reply:
Which would you recommend?

Reply:
I think we should have one one document for the whole section and use anchors to link to specific sections from nav.adoc.